docs/content/en/docs/concepts/architecture/deployment-flow.md

```diff
@@ -0,0 +1,179 @@
+---
+title: Flow of deployment
+linktitle: Flow of deployment
+description: Understand the execution flow of a deployment
+weight: 35
+---
+
+The Keptn Lifecycle Toolkit (KLT) deploys a
+[Kubernetes Workload](https://kubernetes.io/docs/concepts/workloads/)
+by passing through a well-defined execution flow.
+
+The execution flow goes through six main phases:
+
+* Pre-deployments-tasks
+* Pre-deployment-evaluation
+* Deployment
+* Post-deployment-tasks
+* Post-deployment-evaluation
+* Completed
+
+Within each phase,
+all tasks are executed in parallel.
+They are not affected by the order
+in which evaluations and tasks are listed in the
+[KeptnApp](../../yaml-crd-ref/app.md/)
+resource.
```

Comment:
or in the annotations `keptn.sh/pre-tasks` and `keptn.sh/pre-evaluations` (same for post items`

Reply:
done